dashboard/src/components/Header/Menu.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useRef, useState } from "react";
+import { Link } from "react-router-dom";
+
+import {
+  angleIcon,
+  applicationsIcon,
+  ClarityIcons,
+  clusterIcon,
+  fileGroupIcon,
+  heartIcon,
+} from "@clr/core/icon-shapes";
+import { CdsButton, CdsIcon } from "../Clarity/clarity";
+import useOutsideClick from "../js/hooks/useOutsideClick/useOutsideClick";
+
+import { IClustersState } from "../../reducers/cluster";
+import Row from "../js/Row";
+
+import { app } from "shared/url";
+import helmIcon from "../../icons/helm-white.svg";
+import operatorIcon from "../../icons/operator-framework-white.svg";
+import "./Menu.css";
+
+ClarityIcons.addIcons(clusterIcon, fileGroupIcon, angleIcon, applicationsIcon, heartIcon);
+
+export interface IContextSelectorProps {
+  clusters: IClustersState;
+  defaultNamespace: string;
+  appVersion: string;
+  logout: () => void;
+}
+
+function Menu({ clusters, defaultNamespace, appVersion, logout }: IContextSelectorProps) {
+  const [open, setOpen] = useState(false);
+  const currentCluster = clusters.clusters[clusters.currentCluster];
+  const namespaceSelected = currentCluster.currentNamespace || defaultNamespace;
+  // Control when users click outside
+  const ref = useRef(null);
+  useOutsideClick(setOpen, [ref], open);
+
+  const toggleOpen = () => setOpen(!open);
+
+  return (
+    <>
+      <div
+        className={`dropdown kubeapps-align-center kubeapps-menu ${open ? "open" : ""}`}
+        ref={ref}
+      >
+        <button
+          className="kubeapps-nav-link"
+          onClick={toggleOpen}
+          aria-expanded={open}
+          aria-haspopup="menu"
+        >
+          <Row>
+            <CdsIcon size="lg" shape="applications" solid={true} />
```

Comment:
This icon color is not correct. You should use `#fafafa`. This change is already covered in the standard `Header` CSS code, not sure if you're using it here.